src/transport/MessageCounterManagerInterface.h

```diff
@@ -0,0 +1,58 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <transport/PeerConnectionState.h>
+#include <transport/SecureSessionHandle.h>
+
+namespace chip {
+namespace Transport {
+
+class MessageCounterManagerInterface
+{
+public:
+    virtual ~MessageCounterManagerInterface() {}
+
+    virtual bool IsSyncCompleted(PeerMessageCounter & counter)                                        = 0;
+    virtual CHIP_ERROR VerifyCounter(PeerMessageCounter & counter, const PacketHeader & packetHeader) = 0;
+    virtual void CommitCounter(PeerMessageCounter & counter, const PacketHeader & packetHeader)       = 0;
```

Comment:
These all need to be documented.....

Reply:
These functions can be remove along with the dummy implementation.